Avoid installing headers and shared library into the py3-mupdf subpackage, as
we already install them in the mupdf-dev and mupdf-libs subpackage.
--- a/Makefile	2025-05-05 21:39:26.970958150 +0200
+++ b/Makefile	2025-05-05 21:40:54.580448730 +0200
@@ -660,7 +660,7 @@ ifneq ($(OS),OpenBSD)
 	ln -sf libmupdfcpp.$(SO)$(SO_VERSION) $(DESTDIR)$(libdir)/libmupdfcpp.$(SO)
 endif
 
-install-shared-python: install-shared-c++ python
+install-shared-python: python
 	install -d $(DESTDIR)$(pydir)/mupdf
 	install -m $(SO_INSTALL_MODE) $(OUT)/_mupdf.$(SO) $(DESTDIR)$(pydir)/mupdf
 	install -m 644 $(OUT)/mupdf.py $(DESTDIR)$(pydir)/mupdf/__init__.py
